server/index.js

```diff
@@ -2,15 +2,38 @@ require('dotenv/config');
 const express = require('express');
```

Comment:
Do you want to use import instead of require? Optional.
If you do, you need to add `"type": "module"` to your package.json.

Reply:
Hrm. What would be the benefit of switching to import here?

Reply:
Nothing. It's just more modern. I'm switching the exercises to import, but I'm finding there are gotchas along the way. So maybe it's best to just stick with `require` for now.